prac_09/cleanup_files.py

```diff
@@ -0,0 +1,38 @@
+"""
+CP1404/CP5632 Practical
+Demos of various os module examples
+"""
+
+import os
+
+
+def main():
+    """Process all subdirectories using os.walk()."""
+    os.chdir('Lyrics')
+    for directory_name, subdirectories, filenames in os.walk('.'):
+        print("Directory:", directory_name)
+        print("\tcontains subdirectories:", subdirectories)
+        print("\tand files:", filenames)
+        print("(Current working directory is: {})".format(os.getcwd()))
+        for name in filenames:
+            old_name = os.path.join(directory_name, name)
+            new_name = os.path.join(directory_name, get_fixed_filename(name))
+            print(new_name)
+            os.rename(old_name, new_name)
+
+
+def get_fixed_filename(filename):
+    """Return a 'fixed' version of filename."""
+    new_name = filename.replace(".TXT", ".txt")
```

Comment:
Possibly change to `filename = filename.replace(".TXT", ".txt")` as it can get confused with new_text and I don't think a new variable would be needed in this case. Not certain though and mainly personal preference so it's up to you whether you want to add it